packages/usa-combo-box/src/styles/_usa-combo-box.scss

```diff
@@ -56,13 +56,25 @@ $close: map-merge(
   margin-bottom: 0;
   max-width: none;
   padding-right: calc(2.5em + 3px);
+
+  &:disabled,
+  &[aria-disabled="true"] {
+    background-color: color("disabled");
+    cursor: not-allowed;
+  }
```

Comment:
I see these are rules repeated a lot. Could we create a mixin for this?

Reply:
Oh! I didn't even consider that! I like that idea!